### ISSUE TITLE
oobabooga amd gpu windows

### ISSUE THREAD

**Issue by F1re4** [oobabooga/text-generation-webui#6525]
I've searched the entire Internet, I can't find anything; it's been a long time since the release of oobabooga. There are ways to run it on an AMD GPU RX6700XT on Windows without Linux and virtual environments. When I try, it just says that PyTorch is not fully installed, but nothing helps, tell me Please

**Comment by RSAStudioGames** [comment]
Do you have ROCM installed? If I remember correctly, ROCM only works on baremetal linux. Not even WSL unfortunately.
https://rocm.docs.amd.com/projects/install-on-linux/en/latest/how-to/native-install/ubuntu.html

AMD's docs has all the install info you need.

**Comment by F1re4** [comment]
so u saying its only possible with linux main os, on windows it is not possbible with Windows?

**Comment by F1re4** [comment]
What is the best alternative for AMD GPU, local, many models, very simillar to Oobabooga but for AMD GPU?

**Comment by RSAStudioGames** [comment]
So I just looked it up, seems like there is a HIP SDK from ROCM for windows 10/11. I haven't tried it so I can't guarantee anything, but that could be a possible option to get Oobabooga working.
https://www.amd.com/en/developer/resources/rocm-hub/hip-sdk.html

An alternative to Oobabooga would be:
https://github.com/YellowRoseCx/koboldcpp-rocm
I used this with my 7900XT at one point on windows. I believe it only runs GGUF/GGML models though. That shouldn't be an issue though as a lot of people convert models to GGUF and it also lets you run models using your RAM and VRAM combined.

Hope that helps. :)
